src/hooks/useQuizState/useQuizApiState.ts

```diff
@@ -75,6 +90,13 @@ const useQuizApiState: UseQuizApiState = (
     }
   };
 
+  useEffect(() => {
+    (async () => {
+      dispatchQuizResultsConfig();
+    })();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
```

Comment:
Do we need to wrap dispatchQuizResultsConfig in an async function here?

Since we're just calling it I believe we can just do right?
```
useEffect(() => {
  dispatchQuizResultsConfig();
}, []);
```

Reply:
@v-i-s-h-n-u-ps Sorry I didn't mention this from the start but looking further into this I'm thinking we need to call `dispatchQuizResultsConfig();` where we call `dispatchQuizResults();` instead of calling it in its own effect (like screenshot 1). This way both of them are called/updated together. The only problem I faced was that `getResultsConfig` get's called twice. The same problem happens with getSearchResults as well but we don't see the second API request since it throws an error (screenshot 2)

Can we call both dispatchQuizResultsConfig and dispatchQuizResults together and fix the problem that both are being called twice? I know the duplicate requests issue isn't related to this PR so please let me know If you need any help here.

<img width="603" alt="Screenshot 2023-12-04 at 6 52 58 PM" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/58053149/be19e3ea-2eb2-4043-afe3-719755dd6665">

----------------------------------

<img width="690" alt="Screenshot 2023-12-04 at 6 55 41 PM" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/58053149/ea332284-27c6-4178-ae93-791f3f85fbd8">
)

Reply:
As discussed offline, leaving it in a separate `useEffect`.